src/lib/stitching/gravity/stitching.ts

```diff
@@ -8,12 +9,35 @@ export const gravityStitchingEnvironment = (localSchema: GraphQLSchema) => {
       extend type ViewingRoomArtwork {
         artwork: Artwork
       }
-    `,
 
+      extend type ViewingRoom {
+        artworks: ArtworkConnection
+      }
+    `,
     resolvers: {
+      ViewingRoom: {
+        artworks: {
+          resolve: (parent, _args, context, _info) => {
+            const ids = parent.artworksConnection.edges.map(
```

Comment:
How are pagination arguments going to flow from `artworks` (`first`, `last`, `after`, ...) through to the underlying Gravity field?

I would switch the Gravity schema to add `artworkIDs` as a field on the viewing room parent. That field should be the full, un-paginated list of artwork id's.

Then, you can slice into that field using the pagination arguments, and call `artworksLoader`.